components/src/atoms/index.ts

```diff
@@ -10,3 +10,5 @@ export * from './StepMeter'
 export * from './StepMeter'
 export * from './StyledText'
 export * from './Tag'
+export * from './Tooltip'
+export * from './DropdownMenu'
```

Comment:
nit: this can be alphabetical
```suggestion
export * from './DropdownMenu'
export * from './StepMeter'
export * from './StyledText'
export * from './Tag'
export * from './Tooltip'
```

Reply:
I changed the directory from `atoms` to `molecules` since `DropdownMenu` uses multiple atom components